src/main/java/org/breedinginsight/brapps/importer/services/processors/ExperimentProcessor.java

```diff
@@ -82,10 +80,7 @@
 public class ExperimentProcessor implements Processor {
 
     private static final String NAME = "Experiment";
-    private static final String MISSING_OBS_UNIT_ID_ERROR = "Experiment Units are missing Observation Unit Id.<br/><br/>" +
-            "If you’re trying to add these units to the experiment, please create a new environment" +
-            " with all appropriate experiment units (NOTE: this will generate new Observation Unit Ids " +
-            "for each experiment unit).";
+    private static final String MISSING_OBS_UNIT_ID_ERROR = "Error(s) detected in file. (See details below.) Import cannot proceed.";
```

Comment:
```suggestion
    private static final String MISSING_OBS_UNIT_ID_ERROR = "Experimental entities are missing ObsUnitIDs. You must append file with the appropriate ObsUnitIDs or specify a new environment for these entities to proceed";
```

Per Shawn's comment: https://breedinginsight.atlassian.net/browse/BI-1761?focusedCommentId=14200

As is, the user sees a message with duplicate text and no specific information. 
![image](https://github.com/Breeding-Insight/bi-api/assets/128052931/ab036383-214c-4236-924a-33174f6b84b0)